applications/openshift/master/file_groupowner_kube_apiserver/rule.yml

```diff
@@ -1,15 +1,15 @@
 documentation_complete: true
 
-prodtype: ocp4
+prodtype: rhcos4
 
 title: 'Verify Group Who Owns The Kube API Server Pod Specification File'
 
-description: '{{{ describe_file_group_owner(file="/etc/kubernetes/static-pod-resources/kube-apiserver-pod.yaml", group="root") }}}'
+description: '{{{ describe_general_file_group_owner(filedescription="API server pod specification", group="root") }}}'
 
 rationale: |-
-    The Kube specification file contains information about the configuration of the
-    OpenShift API Server that is configured on the system. Protection of this file is
-    critical for OpenShift security.
+    The API server pod specification file controls various parameters that set the behavior
+    of the API server. You should set its file ownership to maintain the integrity of the file.
+    The file should be owned by root:root.
```

Comment:
@JAORMX these rationale changes don't make much sense in some cases. For example: `The file should be owned by root:root.` That statement should be in the OCIL and/or the description. The rationale is for actually talking about why you need this rule. The original rationale makes more sense because of this.